Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1162,6 +1164,7 @@ def lhegensim(fragment,howMuch):
 steps['RECODR2_25nsreHLT']=merge([{'--hltProcess':'reHLT'},steps['RECODR2_25ns']])
 steps['RECODR2_50nsreHLT']=merge([{'--hltProcess':'reHLT'},steps['RECODR2_50ns']])
 steps['RECODR2_2016reHLT']=merge([{'--hltProcess':'reHLT','--conditions':'auto:run2_data_relval'},steps['RECODR2_2016']])
+steps['RECODR2newL1repack_2016reHLT']=merge([{'-s':'L1REPACK:FullSimTP,RAW2DIGI,L1Reco,RECO,EI,PAT,ALCA:SiStripCalZeroBias+SiStripCalMinBias+TkAlMinBias+EcalESAlign,DQM:@standardDQM+@miniAODDQM','--hltProcess':'reHLT'},steps['RECODR2_2016']])
```

Comment:
@rekovic 
steps['RECODR2_2016'] contains already '--conditions':'auto:run2_data_relval'

https://github.com/cms-l1t-offline/cmssw/blob/a2b4491746fdd457a8fffc51395a36b9d900004e/Configuration/PyReleaseValidation/python/relval_steps.py#L1030

In steps['RECODR2newL1repack_2016reHLT'] it is not overwritten in the merge command, so we have the same conditions as in steps['RECODR2_2016'].
This is OK, I'ld say.